<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color : #c7ccb9;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>551</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Choose file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Screen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QComboBox{
  color: #364330;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Saphire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  color: #364330;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Solid</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Projection_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Vertexes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Projection">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Projection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Vertex_Color">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
	color: white;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}
</string>
    </property>
    <property name="text">
     <string> color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>600</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>600</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	qproperty-alignment: 'AlignVCenter | AlignRight';
  	border: 1px solid #364330;
  	color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Edges_Color">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
	color: white;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}
</string>
    </property>
    <property name="text">
     <string> color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Projection_Color">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
	color: white;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}

QPushButton::Pressed{
	background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string> color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  color: #364330;
}</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>630</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  color: #364330;
}</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Edges</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollBar" name="Scroll_ShiftZ">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>490</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar{
	background-color: rgba(0, 0, 0, 77);

}

QScrollBar::add-line:horizontal {
      border: none;
      background: none;
}

QScrollBar::sub-line:horizontal {
      border: none;
      background: none;
}</string>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  color: #364330;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Central</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parallel</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>640</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #B0BC98;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
     font: bold 14px;
     padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Gif</string>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>251</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid #364330;
  color: #364330;
	border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #DBF9B8;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>320</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vectors = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Shift_X">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>431</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
	background-color : #c7ccb9;
color:rgb(0, 0, 0);
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Shift_Y">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
	background-color : #c7ccb9;
color:rgb(0, 0, 0);
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Shift_Z">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>494</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
	background-color : #c7ccb9;
color:rgb(0, 0, 0);
border:none;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <zorder>Projection_8</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>Projection_2</zorder>
   <zorder>Projection</zorder>
   <zorder>Vertex_Color</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>horizontalScrollBar_6</zorder>
   <zorder>Projection_5</zorder>
   <zorder>Edges_Color</zorder>
   <zorder>label_4</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>horizontalScrollBar_7</zorder>
   <zorder>horizontalScrollBar_2</zorder>
   <zorder>horizontalScrollBar_5</zorder>
   <zorder>Projection_7</zorder>
   <zorder>Projection_4</zorder>
   <zorder>Projection_6</zorder>
   <zorder>label_2</zorder>
   <zorder>horizontalScrollBar_3</zorder>
   <zorder>label_3</zorder>
   <zorder>label_7</zorder>
   <zorder>Projection_3</zorder>
   <zorder>Scroll_ShiftZ</zorder>
   <zorder>comboBox_4</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>doubleSpinBox</zorder>
   <zorder>doubleSpinBox_2</zorder>
   <zorder>label</zorder>
   <zorder>label_8</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>Projection_Color</zorder>
   <zorder>Shift_X</zorder>
   <zorder>Shift_Y</zorder>
   <zorder>Shift_Z</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
